CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ The following changes have been implemented but not released yet:
 
 ## [Unreleased]
 
+### Changed
+
+- `getUrl` now returns local URLs (e.g. #my-thing)
+- `getThing` now returns empty Things if they were added, rather than `null`.
+- The internal structure of SolidDatasets has been reworked. This shouldn't
+  affect you if you relied on just our public API's, but if you were
+  manipulating the data directly, you might need to update to use our public
+  API's. If something's not covered that you need, please [file an
+  issue](https://github.com/inrupt/solid-client-js/issues/new/choose).
+
+### New features
```

Comment:
Mutability didn't work already (i.e. would break as soon as you tried to save the SolidDataset), and for that reason RDF/JS compatibility was never advertised. Since breaking changes only concern public API's, this particular change is not breaking in the SemVer sense that it warrans a major version bump.

The changes listed under "Changes" _can_ theoretically be considered breaking changes. However, since major version bumps are intended to communicate to downstreams that they'll have to pay extra attention to the new release and carefully read the release notes for potential migration paths, and since as far as I'm aware nobody will _actually_ need to make any code changes to accommodate those changes, I think we should not treat them as such.

Reply:
(so apparently you cannot remove a reaction emoji, even in case of misclick ^^)

Reply:
Are you sure? I just removed one by clicking the reaction itself.

Reply:
There must have been a bug: I clicked on the +1, it removed both ^^